.github/workflows/test.yml

```diff
@@ -36,6 +36,11 @@ jobs:
           - target: wasm32-wasi
             os: ubuntu-latest
             minimal_setup: true
+            toolchain: "1.74"
+          - target: wasm32-wasip1
+            os: ubuntu-latest
+            minimal_setup: true
+            toolchain: "stable"
```

Comment:
According to https://blog.rust-lang.org/2024/04/09/updates-to-rusts-wasi-targets.html.